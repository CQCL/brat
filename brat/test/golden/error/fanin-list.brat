f(Bool, Bool) -> List(Bool)
f = { [\/] }
