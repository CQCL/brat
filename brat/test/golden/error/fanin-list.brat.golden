Error in test/golden/error/fanin-list.brat@2:5
f = { [\/] }
    ^^^^^^^^

  Type error: Fanin ([\/]) only applies to Vec

